hack/verify-internal-modules.sh

```diff
@@ -31,14 +31,14 @@ mkdir -p "${_tmp_hack}/.."
 cp -a "${KUBE_ROOT}/hack" "${_tmp_hack}/.."
 
 pushd "${_tmp_kuberoot}" >/dev/null
-./hack/update-hack-tools.sh
+./hack/update-internal-modules.sh
 popd
 
 diff=$(diff -Naupr "${KUBE_ROOT}/hack" "${_tmp_kuberoot}/hack" || true)
```

Comment:
this only diffs the `hack` dir, but `hack/update-internal-modules.sh` now modifies stuff under `hack` and `staging`

Reply:
doh, will fix

Reply:
Will fix by switching to use git worktree - faster AND more complete.